### ISSUE TITLE
RS2 and RS8 not showing from headset

### ISSUE THREAD

**Issue by nomadddamon** [praydog/REFramework#108]
For both of the games I followed the directions of putting the zip file into the install folder, then extracted the zip in the install folder. 

When I load up both games they do not load into the headset, instead just desktop view. I cant get the game to show up in the headset. I am using the valve index. Steam overlay and theater mode are disabled for both games

Any help is appreciated. Thanks!

**Comment by anzbert** [comment]
try this: https://www.reddit.com/r/AnthemTheGame/comments/ar5b8b/psa_workaround_to_disable_hdr_on_nvidia_cards/

or unplug your monitor. i think its an issue with HDR screens

**Comment by daemonatheart** [comment]
Just to add to this, both games give this in the logs for me:

[2022-01-01 22:16:05.131] [REFramework] [info] VR: Unknown event: 412
[2022-01-01 22:16:05.131] [REFramework] [info] VR: Unknown event: 412
[2022-01-01 22:16:05.131] [REFramework] [info] VR: Unknown event: 1306
[2022-01-01 22:16:05.131] [REFramework] [info] VR: Unknown event: 1703
[2022-01-01 22:16:05.131] [REFramework] [info] VR: Unknown event: 1702
[2022-01-01 22:16:05.131] [REFramework] [info] VR: Unknown event: 1702
[2022-01-01 22:16:05.131] [REFramework] [info] VR: Unknown event: 409
[2022-01-01 22:16:05.131] [REFramework] [info] VR: Unknown event: 1703
[2022-01-01 22:16:05.131] [REFramework] [info] VR: Unknown event: 1701
[2022-01-01 22:16:05.131] [REFramework] [info] VR: Unknown event: 1701
[2022-01-01 22:16:05.131] [REFramework] [info] VR: Unknown event: 1702
[2022-01-01 22:16:05.131] [REFramework] [info] VR: Unknown event: 1702
[2022-01-01 22:16:05.131] [REFramework] [info] VR: Unknown event: 409
[2022-01-01 22:16:05.131] [REFramework] [info] VR: Unknown event: 412
[2022-01-01 22:16:05.131] [REFramework] [info] VR: Unknown event: 1707
[2022-01-01 22:16:05.131] [REFramework] [info] VR: Unknown event: 1707
[2022-01-01 22:16:05.131] [REFramework] [info] VR: Unknown event: 1707
[2022-01-01 22:16:05.131] [REFramework] [info] VR: Unknown event: 1707
[2022-01-01 22:16:05.131] [REFramework] [info] VR: Unknown event: 1707

**Comment by GyBrim** [comment]
Same. RE2 and RE3 don't show in my index headset. I can only hear the game sound, it's still SteamVR void environment. I've tried the control panel disable hdr fix, fullscreen and windowed mode, SteamVR beta and stable versions. There's nothing more I can think of except rolling back to windows 10? I'm using windows 11 maybe that's the problem I don't know.

**Comment by nomadddamon** [comment]
> try this: https://www.reddit.com/r/AnthemTheGame/comments/ar5b8b/psa_workaround_to_disable_hdr_on_nvidia_cards/
> 
> or unplug your monitor. i think its an issue with HDR screens

So I tried that as well as just unplugging the monitor, but it still didn't work.

Something else is that the ms on my index doesnt go up at all when I run the rs2 or rs8, and sits at .5ms . But when I run games like GTFO or Minecraft in VR it goes up to like 6.5ms

**Comment by daemonatheart** [comment]
> Same. RE2 and RE3 don't show in my index headset. I can only hear the game sound, it's still SteamVR void environment. I've tried the control panel disable hdr fix, fullscreen and windowed mode, SteamVR beta and stable versions. There's nothing more I can think of except rolling back to windows 10? I'm using windows 11 maybe that's the problem I don't know.

I'm on windows 10 so I don't think that's it...

**Comment by nomadddamon** [comment]
OK! So I fixed my issue somehow. I have an rtx3080 so I used GeForce experience to launch RS6 with steam vr already running just to see if that did anything. As usual it didn't show up, so I walked over to my desktop and "alt+tabed" out of the game(my settings for the game are set to full screen). When I did that, the "ms" on my headset shot up to like 150 and the game crashed. I started the game up again and now it works!!!!!

So far, I closed the game and re-opened a couple times to make sure it would keep working, and what I discovered is the game starts up on the desktop in full screen with nothing showing up in VR. Then after REframework is initialized, I use the menu on my Index to look at the desktop veiw, and the game view "snaps" into place. Even RS2 is working now with the same method. What's weird is that i toggled from checking the vr environment, to looking at vr desktop view many many time over the weekend. It wasn't until after the "alt-tabb" crash today that the method is doing anything. Very bizarre, but its working! 


Also, i am running dx12 yet its still working.
